xfel/conda_envs/README.md

```diff
@@ -0,0 +1,65 @@
+# Psana Build Instructions
+
+There are two environments in this directory. Psana_environment.yml is suitable
+for general use and contains the usual CCTBX dependencies plus psana and its
+dependencies. Psana_lcls_environment.yml should only be used at LCLS; it adds
+a local build of mpi4py that is required for running jobs on multiple psana
+nodes.
+
+The build steps below were tested on Oct 28, 2020. They should be done in a
+clean environment (no cctbx installation has been activated, etc.)
+
+Creating the conda environment (step `base`) may take up to ~20 min with no
```

Comment:
We can change `install_conda.py` to show output as it's happening. It is currently using the `check_output` function which stores the output (and the `--verbose` flag for `libtbx.install_conda` will display it at the end), but we can switch it to `call` which will display the output while processing.

Reply:
That would be nice and would catch some problems early. E.g. when the -y flag was missing, I waited about an hour for something to happen...